stdlib/3.7/dataclasses.pyi

```diff
@@ -0,0 +1,67 @@
+from typing import overload, Any, Callable, Dict, Generic, Iterable, Mapping, Optional, Tuple, Type, TypeVar, Union
+
+
+_T = TypeVar('_T')
+
+_DictFactory = Callable[List[Tuple[str, Any]], _T]
```

Comment:
This is not quite right; you need one more set of [] around the first argument to Callable, because that's how you indicate the argument list.